cypress/integration/index.spec.js

```diff
@@ -0,0 +1,65 @@
+/// <reference types="cypress" />
+describe("계산기 테스트", () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:5000/');
+    });
+
+    it('계산기가 존재한다.', () => {
+        cy.get('.calculator').should('exist')
+    });
+    it('2개의 숫자에 대해 덧셈이 가능하다.', () => {
+        cy.get('button:contains("1")').click()
+        cy.get('button:contains("+")').click()
+        cy.get('button:contains("2")').click()
+        cy.get('button:contains("=")').click()
```

Comment:
이런식으로도 셀렉터를 실행할 수 있군요 ㅎㅎ 처음 알았습니다

Reply:
```js
```suggestion
        handleClickNumber(1);
        handleClickPlus();
        handleClickNumber(2);
        handleClickEquals();
```
이렇게 함수로 표현해주면 테스트코드를 작성할 때 더 수월하지 않을까요!?